sys/include/net/ng_fib/ng_universal_address.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2014  Martin Landsmann <Martin.Landsmann@HAW-Hamburg.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_universal_address Universal Address Container
+ * @ingroup     net
+ * @brief       universal address container
+ *
+ * @{
+ *
+ * @file
+ * @brief       Types and functions for operating universal addresses
+ * @author      Martin Landsmann
+ */
+
+#ifndef UNIVERSAL_ADDRESS_H_
+#define UNIVERSAL_ADDRESS_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define UNIVERSAL_ADDRESS_SIZE (16)         /**< size of the used addresses in bytes        */
+#define UNIVERSAL_ADDRESS_SUCCESS (0)       /**< return value for success                   */
```

Comment:
We wouldn't need that if https://github.com/RIOT-OS/RIOT/pull/2211/files#r27515463 is addressed